src/containers/header/Header.jsx

```diff
@@ -66,30 +66,35 @@ export default class Header extends React.Component {
         const { movieData } = this.props;
         const { resultsQuantity } = this.state;
 
-        const content = !movieData ?
-            <SearchForm
-                onSearchBarChange={ this.handleSearchBarChange }
-                onSearchByChange={ this.handleSearchByChange }
-                onSearchButtonClick={ this.handleSearchButtonClick }
-                searchBy={ this.state.searchBy } />
-            : <MovieDetails  { ...movieData } />;
         const genre = movieData && movieData.genres && movieData.genres[0];
 
         return (
             <div className="header">
                 <div className="header-content">
-                    <Logo />
-
-                    {
-                        movieData &&
-                        <Button
-                            className="button search-return-button"
-                            content="Search"
-                            onClickHandler={ this.handleSearchReturnClick }
-                        />
+                    <div className="row">
+                        <div className="col-xs">
+                            <Logo />
+                        </div>
+
+                        { movieData &&
```

Comment:
This component could be split into two, the content of header could be applied in the upper level and you can use `children` prop to make it decoupled.

```javascript

// header file
export default const Header = () => (
    <div className="header">
        <div className="header-content">
             <div className="row">
                  <div className="col-xs"> <Logo /> </div>
                  {this.props.children}
             </div>
        </div>
    </div> 
);

// where you want to use Header file
...
render() {
    return (
       <Header>
             {showSearch && <SearchForm /> }
             {showMovie && <MovieBanner />}
       </Header>
    );
}
...